beginend.el

```diff
@@ -225,16 +225,41 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+(add-to-list 'beginend-modes (cons 'mu4e-view-mode-hook #'beginend-message-mode))
```

Comment:
Also format it like so:

```
(
 (...)
 (...)
 )
```

This looks weird but actually is "acceptable" in `defvar`s and constants as it produces better diffs when lines are added/removed.